ragna/deploy/_ui/central_view.py

```diff
@@ -613,26 +613,27 @@ def header(self):
 
                 chat_documents_pills.append(pill)
 
-        chat_info_button = None
-        if self.current_chat is not None:
-            button_name = f"{self.current_chat['metadata']['assistant']} | {self.current_chat['metadata']['source_storage']}"
-            chat_info_button = pn.widgets.Button(
+        button_name = f"{self.current_chat['metadata']['assistant']} | {self.current_chat['metadata']['source_storage']}"
+        if hasattr(self, "chat_info_button"):
```

Comment:
Would it make sense to create this button in the constructor with an empty name at first and inside this method unconditionally overwrite the name?